.github/actions/changelogs-generator/action.yml

```diff
@@ -0,0 +1,59 @@
+name: Generate Changelogs
```

Comment:
What is advantage now of making a separate "actions" yml file?  We don't have a `.github./actions` directory yet.

Reply:
I have added a custom GitHub Action to be used by the [workflow template](https://github.com/edx/.github/blob/jawayria/semantic-release/.github/workflows/changelogs.yml). I had to write a separate action because I wanted to call a python script [changelogs.py](https://github.com/edx/.github/blob/jawayria/semantic-release/.github/actions/changelogs-generator/changelogs.py) to generate changelogs and if I called it directly from the workflow template, I had to copy the script in every repository. 
Here I wrote a custom action and included the script there so I don't need to replicate it to all the repos now.

Reply:
Is that something a reusable workflow could do now?  I'm just trying to limit the number of different ways we do things.

Reply:
Will check the latest docs

Reply:
This is still not supported as mentioned in the [GitHub Docs](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables). See description for `GITHUB_ACTION_PATH`.